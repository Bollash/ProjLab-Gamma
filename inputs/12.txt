Create Space
Add Asteroid Iron
Add Settler 0
Add TpGate Settler 0 Settler 0
Add Material 0 Iron
Add Material 0 Iron
Add Material 0 Ice
Add Material 0 Uran
Build TpGate 0
Status All