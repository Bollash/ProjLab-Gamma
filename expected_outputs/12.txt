Space létrehozva
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
Sikeres volt a telepeshozzáadás
Hozzáadtuk a teleportkapukat a célpontokhoz.
Sikeresen betettük a nyersanyagot a telepesbe
Sikeresen betettük a nyersanyagot a telepesbe
Sikeresen betettük a nyersanyagot a telepesbe
Sikeresen betettük a nyersanyagot a telepesbe
Nem sikerült a teleportkapu építés, mert nincs hely.
Space
1
10
10
False
1
1
Asteroid
0
3
3
3
Iron
0
1
0
Settler
0
0
0
0
Iron
Iron
Ice
Uran